boost_adaptbx/tests/tst_deprecation_tools.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, division, print_function
+from boost_adaptbx.deprecation_tools import deprecate_method
+import warnings
+
+
+def exercise():
+  with warnings.catch_warnings(record=True) as w:
+    warnings.simplefilter("always")
+
+    # Choose a Boost Python class for which to deprecate a method. Here we
+    # use rational.int.
+    from boost_adaptbx.boost import rational
+
+    original_value = rational.int().numerator()
+    deprecate_method(rational.int, "numerator")
```

Comment:
```suggestion
  # Choose a Boost Python class for which to deprecate a method. Here we
  # use rational.int.
  from boost_adaptbx.boost import rational

  original_value = rational.int().numerator()
  deprecate_method(rational.int, "numerator")
    
  with warnings.catch_warnings(record=True) as w:
    warnings.simplefilter("always")
```

to ensure the warning is thrown at call-time, not at setup-time.

Reply:
Ok, can do. I was wrapping the first call to numerator too in order to be sure that didn't already raise a warning, but I suppose that isn't really necessary.